packages/cli/src/server.ts

```diff
@@ -263,6 +263,17 @@ export class Server extends AbstractServer {
 			res.sendFile(tzDataFile, { dotfiles: 'allow' }),
 		);
 
+		// Serve third-party licenses file
+		const licenseFile = resolve(CLI_DIR, 'dist', 'THIRD_PARTY_LICENSES.md');
+		this.app.get('/THIRD_PARTY_LICENSES.md', async (_, res) => {
```

Comment:
Since this file can reveal details about the n8n instance, it would be good to have this behind auth. We can do it by moving this to a `ThirdPartyLicensesController`. See [`ModuleSettingsController`](https://github.com/n8n-io/n8n/blob/include-third-party-licenses-md-with-release/packages/cli/src/controllers/module-settings.controller.ts#L16-L17) for reference